pyproject.toml

```diff
@@ -1,26 +1,31 @@
 [project]
-name = "mlops-with-databricks"
+name = "wine-quality"
 version = "0.0.1"
-description = "MLOps with Databricks"
+description = "MLOps with Databricks using Wine Quality"
 readme = "README.md"
 requires-python = ">=3.11, <3.12"
 dependencies = ["lightgbm>=4.5.0, <5",
-                "scikit-learn>=1.5.1, <2",
-                "cloudpickle>=3.0.0, <4",
-                "mlflow>=2.16.0, <3",
+                "scikit-learn>=1.5.2, <2",
+                "cloudpickle>=3.1.0, <4",
+                "mlflow>=2.17.0, <3",
                 "numpy>=1.26.4, <2",
-                "pandas>=2.2.2, <3",
-                "pyarrow>=15.0.2, <16",
+                "pandas>=2.2.3, <3",
+                "pyarrow>=14.0.1",
```

Comment:
this is problematic, it must be exactly that version, otherwise you may get problems with model serving.
The same goes for the rest of the packages.

Reply:
Updated for week 3 feature